### PR TITLE
Fix css for profile active

### PR DESCRIPTION
**Pull request by Ian-Yy**
##### Description

I've modify some class used in the elements when generating the profile/active screen.
Added css for a new class too.

##### Refers/Fixes

This PR is to fix Issue #5291 

##### Testing

I've went through the contributors guide and I don't think my changes need to go through Django Translation Framework. I've checked the screen too.

Please tell me if the screen is not to your liking.

### CODE REVIEW

File: app/dashboard/templates/profiles/tab_active.html [hunk 1]
```diff
@@ -2,20 +2,22 @@
...
-        <small class="align-self-center">Bounty opened {{bounty.web3_created|naturaltime}} for {{bounty.org_name}}</small>
+        <div class="info font-caption">Bounty opened {{bounty.web3_created|naturaltime}} for {{bounty.org_name}}</div>
         <div class="tag-list">
           {% for keyword in bounty.keywords_list %}
             <small class="tag-list__item">{{keyword}}</small>
           {% endfor %}
         </div>
       </div>
-      <div class="float-right">
+      <div class="col-12 col-md-4 col-lg-3 font-caption bounty-button">
```
Comment (thelostone-mc) [inline app/dashboard/templates/profiles/tab_active.html]: `bounty-button` -> `my-auto text-right`

File: app/assets/v2/css/dashboard.css [hunk 2]
```diff
@@ -149,6 +149,12 @@ body {
   top: 0px;
 }
 
+.bounty-button {
+  text-align: right;
+  margin-top: auto;
+  margin-bottom: auto;
+}
```
Comment (thelostone-mc) [inline app/assets/v2/css/dashboard.css]: ^ we can remove this and use already available classes

**Comment by Ian-Yy** [comment]
Okay. Will do the changes ASAP. Sorry I did not realise there's an existing class doing the same thing. Will change it and will attach a screenshot of the updated UI.

**Comment by Ian-Yy** [comment]
Hi @thelostone-mc , I've removed the new class and css from dashboard.css. Attached below is the screenshot. Is the UI ok? If it's ok, then I will proceed with the commit and create PR. :grinning:

gitcoin

**Comment by thelostone-mc** [comment]
@Ian-Yy looks good ^_^ good you ensure the photos are rounded ? like in the activity section ? we have a class `avatar` ( i think that's what it called ) It adds a border-radius to it ^_^

**Comment by Ian-Yy** [comment]
@thelostone-mc It's not rounded currently. I saw the avatar class, will add it for the border-radius and post another screenshot.

**Comment by thelostone-mc** [comment]
@Ian-Yy go for it and use that class and update this PR ^_^
Let's get this in 🙌

**Comment by Ian-Yy** [comment]
@thelostone-mc OMG. I am sorry I accidentally messed up my branch. Should I create another branch and resubmit another PR or fix this PR? Sorry, I'm still new to these fetching and rebasing. 😭

**Pull request closed without merging**